app/actions/remote/post.ts

```diff
@@ -59,23 +59,33 @@ export const fetchPostsForChannel = async (serverUrl: string, channelId: string)
         // Here we should emit an event that fetching posts failed.
     }
 
+    let authors: UserProfile[] = [];
     if (data.posts?.length && data.order?.length) {
         try {
-            await fetchPostAuthors(serverUrl, data.posts, false);
+            const {authors: fetchedAuthors} = await fetchPostAuthors(serverUrl, data.posts, false);
+            authors = fetchedAuthors || [];
         } catch (error) {
             // eslint-disable-next-line no-console
             console.log('FETCH AUTHORS ERROR', error);
         }
 
-        operator.handlePosts({
-            actionType,
-            order: data.order,
-            posts: data.posts,
-            previousPostId: data.previousPostId,
-        });
+        if (!fetchOnly) {
+            const models = await operator.handlePosts({
+                actionType,
+                order: data.order,
+                posts: data.posts,
+                previousPostId: data.previousPostId,
```

Comment:
On the server side, previousPostId is defined as:
`PrevPostId string           `json:"prev_post_id"``
Therefore, it will always be defined (even if it is just an empty string).

Reply:
Got you, thanks :)